---
int64 obj_type
geometry_msgs/Pose[] push_locations

